src/node_crypto_groups.h

```diff
@@ -394,20 +394,19 @@ typedef struct {
   const char* name;
   const char* prime;
   unsigned int prime_size;
-  const char* gen;
-  unsigned int gen_size;
+  unsigned int gen;
 } modp_group;
 
 static const modp_group modp_groups[] = {
 #define V(var) reinterpret_cast<const char*>(var)
```

Comment:
You don't need this macro anymore if the groups are defined as `const char group_x[]` instead of `const unsigned char group_x[]`

Reply:
I thought so, too, but then the compiler warns about values in the array definitions being out of range (for signed `char`s).